.ci/validation.sh

```diff
@@ -201,7 +201,7 @@ no-error-xwiki)
   cd ..
   checkout_from https://github.com/xwiki/xwiki-platform.git
   cd .ci-temp/xwiki-platform
-  git checkout "b31a47e4f84b1981c27fd55716c""eb89f6983af5b"
+  git checkout "f52fa56cc13fdc64dc80c29a96cd545e023e7121"
```

Comment:
Updated to master. Not sure why specific commit of `xwiki-platform` is checked out. It needs to be updated every time xwiki makes a new release.
[Travis config](https://github.com/checkstyle/checkstyle/commit/fecf3b6d9095bb41173c29ae637604c542a6d825#diff-5b8d3053aff77c4973f507c3526670e22c8100ac44358c1513825365d009cd05L310-L313) was building head.